addons/payment_worldline/models/payment_provider.py

```diff
@@ -65,8 +66,7 @@ def _worldline_make_request(self, endpoint, payload=None, method='POST', idempot
         api_url = self._worldline_get_api_url()
         url = f'{api_url}/v2/{self.worldline_pspid}/{endpoint}'
         content_type = 'application/json; charset=utf-8' if method == 'POST' else ''
-        tz = datetime.timezone(datetime.timedelta(hours=0), 'GMT')
-        dt = datetime.datetime.now(tz).strftime('%a, %d %b %Y %H:%M:%S %Z')  # Datetime in RFC1123.
+        dt = format_date_time(Datetime.now().timestamp())  # Datetime in RFC1123.
```

Comment:
Just to know where it's coming from.
```suggestion
        dt = format_date_time(Datetime.now().timestamp())  # Datetime in locale-independent RFC1123.
```